astropy/tests/runner.py

```diff
@@ -183,6 +183,18 @@ def run_tests(self, **kwargs):
 
         # Don't import pytest until it's actually needed to run the tests
         import pytest
+        from distutils.version import LooseVersion
+
+        pytest_version = LooseVersion(pytest.__version__)
+        if pytest_version >= '3.7':
```

Comment:
```suggestion
        if pytest_version >= LooseVersion('3.7'):
```

Reply:
weird, now I see the buttons to add the suggestion to a commit, but it definitely wasn't there before even after I assigned myself (thus I sent a feedback for the beta feature).